actions/views/posts.js

```diff
@@ -32,9 +32,8 @@ export function editPost(post) {
     };
 }
 
-export function forwardPost(post, channel, message = '') {
-    return async (dispatch, getState) => {
-        const state = getState();
+export function forwardPost(post, channel, state, message = '') {
+    return async (dispatch) => {
```

Comment:
I think this was as intended, it is OK to `getState` in the thunk action,
we need to avoid the `store.getState();` call.

@nevyangelova maybe the ticket mentions the wrong file, it looks to me that the file that needs refactoring is `actions/post_actions.ts`

e.g

https://github.com/mattermost/mattermost-webapp/blob/e949d96a76995b7149d7b467074f99bd40968db7/actions/post_actions.ts#L156

needs to be refactored so state is not passed down as an argument.